src/main/kotlin/com/salesforce/dynamodbv2/mt/backups/MtBackupAwsAdaptor.kt

```diff
@@ -51,11 +51,39 @@ class MtBackupAwsAdaptor {
                 BackupDescription().withBackupDetails(getBackupDetails(mtBackupMetadata)))
     }
 
+    fun getDescribeBackupResult(backupMetadata: TenantBackupMetadata): DescribeBackupResult {
+        return DescribeBackupResult().withBackupDescription(
+                BackupDescription().withBackupDetails(getBackupDetails(backupMetadata))
+        )
+    }
+
+    fun getTenantTableBackupFromArn(backupArn: String): TenantTableBackupMetadata {
+        if (!isTenantTableArn(backupArn)) {
+            throw IllegalArgumentException("$backupArn does not include tenant-table specifier.")
+        }
+        val tenantTableParts = backupArn.split(':')
+        return TenantTableBackupMetadata(tenantTableParts[0], tenantTableParts[1], tenantTableParts[2], tenantTableParts[3])
+    }
+
+    private fun isTenantTableArn(backupArn: String): Boolean = backupArn.split(':').size == 4
+
+    fun getBackupArnForTenantTableBackup(tenantTable: TenantTableBackupMetadata): String =
+            "${tenantTable.s3BucketName}:${tenantTable.backupName}:${tenantTable.tenantId}:${tenantTable.virtualTableName}"
+
+    fun getBackupArnForTenantTableBackup(tenantTable: TenantBackupMetadata): String =
+            "${tenantTable.s3BucketName}:${tenantTable.backupName}:${tenantTable.tenantTable.tenantName}:${tenantTable.tenantTable.virtualTableName}"
```

Comment:
It seems a bit weird that we're returning an s3 arn instead of a dynamodb backup arn, but I guess either way we're not sticking with their format, so it's ok?

Reply:
its not even an s3 arn, its totally its own thing.. I figured the s3 bucket is important information if/when we try restoring backups from one app into another. In the shorter term, its also import to distinguish data backups from metadata backups with the zos change creating and restoring backups from different buckets.

I imagine wanting to add logic at some point in the future of making sure to download the backup from the bucket in the arn, vs what the mt-dynamo backup manager is currently configured to.